.github/actions/run-interop-ping-test/action.yml

```diff
@@ -0,0 +1,78 @@
+name: 'libp2p ping interop test'
+description: 'Run the libp2p ping interoperability test suite'
+inputs:
+  test-filter:
+    description: "Filter which tests to run out of the created matrix"
+    required: false
+    default: ""
+  extra-versions:
+    description: "Space-separated paths to JSON files describing additional images"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        WORK_DIR=$(realpath "${{ github.action_path }}/../../../multidim-interop")
```

Comment:
Does this depend on where the caller checks this out? Or how did you know how many parent dirs to go up to?

Reply:
This depends on where this file is within this repository. I am basically walking up from the `.github` directory here.

Reply:
Got it. That makes sense. Could we leave a comment here saying that?

Reply:
Will do! Technically, we could even hardcode this path but then we'd depend on GitHub actions runner internals.

Reply:
I added the comment

Reply:
> we could even hardcode this path

This is wrong because it wouldn't work with forks of this action. See an example here: https://github.com/libp2p/rust-libp2p/actions/runs/4060621646/jobs/6989891784#step:6:10

We have to use the `github.action_path` variable to account for this dynamic element.